ddp-workspace/projects/ddp-dsm-ui/app.yaml

```diff
@@ -0,0 +1,18 @@
+service: study-manager-ui
+runtime: python37
+
+# This setting will not matter except to prevent default automatic-scaling which prevents disabling
+# earlier versions of this service
+basic_scaling:
+  max_instances: 1
+
+handlers:
+  - url: /(.*\.(css|eot|gz|html|ico|js|map|png|svg|ttf|woff|woff2|jpg|jpeg|gif|png|json|pdf))
+    static_files: dist/\1
+    upload: dist/(.*\.(css|eot|gz|html|ico|js|map|png|svg|ttf|woff|woff2|jpg|jpeg|gif|png|json|pdf))
+    secure: always
+
+  - url: /(.*)
+    static_files: dist/index.html
+    upload: dist/index.html
+    secure: always
```

Comment:
This file is identical to all the other app.yaml files used in all the studies except for the the first line/service.